eclipse-scout-core/src/form/fields/fields.ts

```diff
@@ -175,5 +176,27 @@ export const fields = {
   focusAndInsertText($input: JQuery<HTMLInputElement | HTMLTextAreaElement>, text: string) {
     $input[0].focus();
     $input.document(true).execCommand('insertText', false, text);
+  },
+
+  /**
+   * @returns true if the element supports the readonly attribute
+   */
+  supportsReadOnlyAttribute($element: JQuery<HTMLInputElement | HTMLTextAreaElement>) {
```

Comment:
The attribute is called "readonly" (one word) - maybe reflect that in the method name?